app/mailers/welcome_mailer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class WelcomeMailer < ActionMailer::Base
+  default from: 'scholar@uc.edu'
+  def welcome_email(email)
+    new_user = User.find_by_email(email)
+    return if new_user.nil?
+    if new_user.student?
+      mail(
```

Comment:
@13miciunasl Maybe shorten this method and dry it up a bit by moving the mail() call into a new send_mail method?  Make from/to/subject/template_names instance variables so send_mail won't need to be passed anything.